tests/ui/components/filters/filteroptionscomponent.js

```diff
@@ -233,3 +229,267 @@ describe('filter options component', () => {
     expect(setStaticFilterNodes.mock.calls).toHaveLength(6);
   });
 });
+
+describe('filter options component selected options', () => {
```

Comment:
Can you provide a more descriptive name for this suite of tests? The name doesn't make it very clear what's being tested.

Reply:
updated!